### DISCUSSION TITLE
How do you download a specific flair from a specific subreddit

### DISCUSSION THREAD

**Discussion by Crossoverfan96** [Serene-Arc/bulk-downloader-for-reddit#577] [Q&A]
I know the basics but i don't know how to use --search 'flair:####' at all much less use it with --subreddit in tandem

**Comment by Serene-Arc** [comment]
I'm not quite sure what you're asking. Putting both options in the command should make them work in tandem. The README has examples and links to how to use the search option.

**Reply by Crossoverfan96** [reply]
where in the README do i find the examples and/or links

**Reply by Serene-Arc** [reply]
In the usage and options sections. The README isn't very long.
